README.md

```diff
@@ -156,7 +163,7 @@ linker = "arm-buildroot-linux-musleabihf-gcc"
 2. Dockerfile:
 
 ```
-FROM rust:1.30.1
+FROM rust:1.36.0
```

Comment:
Is there a reason for the version bump? I would prefer it in another pr

Reply:
The README states the minimum Rust version for this crate to be 1.36.0 and when I tested cross building for another MR and I ran into this and hence I though I boy scout it here. But you are right insofar it does not really belong here and I'll move it into a separate MR.

Reply:
Ah yes. It is fine in this case

Reply:
> But you are right insofar it does not really belong here and I'll move it into a separate MR.

Done.